metadata.json

```diff
@@ -1,38 +1,38 @@
-{
-  "name": "puppetlabs-scheduled_task",
-  "version": "1.0.0",
-  "author": "puppetlabs",
-  "summary": "Manage scheduled tasks for Windows Server 2008 and newer operating systems.",
-  "license": "Apache-2.0",
-  "source": "https://github.com/puppetlabs/puppetlabs-scheduled_task.git",
-  "project_page": "https://github.com/puppetlabs/puppetlabs-scheduled_task",
-  "issues_url": "https://tickets.puppet.com/browse/MODULES",
-  "dependencies": [
-
-  ],
-  "operatingsystem_support": [
-    {
-      "operatingsystem": "windows",
-      "operatingsystemrelease": [
-        "2008",
-        "2008 R2",
-        "2012",
-        "2012 R2",
-        "2016",
-        "7",
-        "8",
-        "8.1",
-        "10"
-      ]
-    }
-  ],
-  "requirements": [
-    {
-      "name": "puppet",
-      "version_requirement": ">= 4.9.0 < 7.0.0"
-    }
-  ],
-  "pdk-version": "1.7.1",
-  "template-url": "https://github.com/puppetlabs/pdk-templates",
-  "template-ref": "heads/master-0-g9c815ea"
-}
+{
+  "name": "puppetlabs-scheduled_task",
+  "version": "1.0.0",
+  "author": "puppetlabs",
+  "summary": "Manage scheduled tasks for Windows Server 2008 and newer operating systems.",
+  "license": "Apache-2.0",
+  "source": "https://github.com/puppetlabs/puppetlabs-scheduled_task.git",
+  "project_page": "https://github.com/puppetlabs/puppetlabs-scheduled_task",
+  "issues_url": "https://tickets.puppet.com/browse/MODULES",
+  "dependencies": [
+
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "windows",
+      "operatingsystemrelease": [
+        "2008",
+        "2008 R2",
+        "2012",
+        "2012 R2",
+        "2016",
+        "7",
+        "8",
+        "8.1",
+        "10"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.9.0 < 7.0.0"
+    }
+  ],
+  "pdk-version": "1.8.0",
```

Comment:
FYI it looks like I picked up a newer template @jpogran .... and running on Windows vs OSX changes the line endings.

Still working on the matrix.